Resources/Prototypes/_Harmony/Entities/Clothing/Uniforms/jumpsuits.yml

```diff
@@ -19,3 +19,25 @@
     sprite: _Harmony/Clothing/Uniforms/Jumpsuit/hop_turtle.rsi
   - type: Clothing
     sprite: _Harmony/Clothing/Uniforms/Jumpsuit/hop_turtle.rsi
+
+- type: entity
+  parent: [ClothingUniformBase, BaseRestrictedContraband]
+  id: ClothingUniformJumpsuitWardenBlueHarmony
+  name: Warden's Blue Suit
```

Comment:
Please make the name all lowercase to match the names of other items in the game